src/test/java/edu/gatech/VRDR/MessagingTest.java

```diff
@@ -637,6 +639,19 @@ public void testCreateAckForDeathRecordVoidMessage() {
         assertEquals(voidMessage.getNCHSIdentifier(), ack.getNCHSIdentifier());
         assertEquals(voidMessage.getBlockCount(), ack.getBlockCount());
     }
+	
+    public void testSetBlockCountDeathRecordVoidMessage() {
+	DeathRecordVoidMessage voidMessage = new DeathRecordVoidMessage();
+// 	Integer value = null;
+// 	voidMessage.setBlockCount(value);
+// 	assertEquals(voidMessage.getBlockCount(), null);
+	Integer value = 2;
+	voidMessage.setBlockCount(value);
+	System.out.println("******value******="+value);
```

Comment:
Please don’t add system out println to the tests, only assertions.

Reply:
Mr. Burns, I am still working on the issue with debugging frags. I will notify you again once I am done. Thank you.